public/admin/templates/badminton.tmpl.html

```diff
@@ -16,7 +16,12 @@ <h2>Badminton</h2>
             <input style="text-align: center;"  type="text" placeholder="Match Name" ng-model="badminton.match">
         </div>
     </div>
-
+    <br /><br />
```

Comment:
Any reason for 2 breaks here instead of the 1 that is elsewhere on the template?